fe/fe-core/src/main/java/com/starrocks/ha/StateChangeExecution.java

```diff
@@ -0,0 +1,8 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.ha;
+
+public interface StateChangeExecution {
+    public void transferToLeader(FrontendNodeType newType);
```

Comment:
`newType` must be `LEADER` and better to remove.

Reply:
done